lib/phash/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
 #![no_std]
+#![allow(clippy::result_unit_err)]
```

Comment:
Probably add a comment explaining why this is required?

Reply:
I think this lint could probably just be disabled in xtask.  I don't think it applies to `no_std`.